akka-sample-distributed-data-java/src/multi-jvm/scala/sample/distributeddata/ShoppingCartSpec.scala

```diff
@@ -21,6 +24,9 @@ object ShoppingCartSpec extends MultiNodeConfig {
     akka.loglevel = INFO
     akka.actor.provider = "cluster"
     akka.log-dead-letters-during-shutdown = off
+    akka.actor.serialization-bindings {
+      "sample.distributeddata.ShoppingCart$LineItem" = jackson-cbor
+    }
```

Comment:
this should also be defined in application.conf

Reply:
I tried this and it didn't work ("No configured serialization-bindings for class [sample.distributeddata.ShoppingCart$LineItem]"), so I dropped that commit and didn't investigate further.

Reply:
application.conf isn't loaded automatically by this test.
probably not a good idea to load it here because of the seed-nodes and such, so a copy of that part of the config is ok